module-8/M8/server/config/db.js

```diff
@@ -1,7 +1,10 @@
 import mongoose from 'mongoose';
 import { Sequelize } from 'sequelize';
+import dotenv from 'dotenv';
+dotenv.config()
 
-const sequelize = new Sequelize('postgres://db_user:135256@localhost:5432/portfolio' , {dialect: 'postgres', logging: false});
+console.log(process.env.db_user, process.env.db_psword, process.env.db_host, process.env.db_port, process.env.db_name);
```

Comment:
Much better!

One minor detail: standard format for environment variables will be an all-capitals "constant" format:
like `DB_PASSWORD` instead of `db_psword`.

Remember to change that both in your Env file, and also in the codebase.

Otherwise looks good.